openebench/README.md

```diff
@@ -0,0 +1,26 @@
+# /openebench/
+
+## Uses
+
+This namespace is going to serve different parts of the [OpenEBench](https://openebench.bsc.es) ecosystem.
+
+* PIDs pointing to scientific benchmarking JSON Schemas.
+* PIDs pointing to the different OpenEBench scientific benchmarking concept entries.
+
+In the future, the namespace will also provide mappings for technical monitoring area.
+
+## Contact
+
+This space is administered by members of [INB (Spanish National Bioinformatics Institute)](https://inb-elixir.es) and [Barcelona Supercomputing Center](https://www.bsc.es)
+
+| Person | GitHub user | ORCID |
+--------------------------------
```

Comment:
For this to work as the intended table, this divider must be changed as follows:
```suggestion
|--------|-------------|-------|
```

Reply:
Thanks for the feedback @TallTed ! I'm adding it in the next pull request, when I am adding additional routes